requirements.txt

```diff
@@ -2,3 +2,4 @@ pytest
 pytest-html
 pytest-cov
 flake8
+pyopenssl
```

Comment:
Should this be removed?

Reply:
https://setuptools.readthedocs.io/en/latest/setuptools.html#declaring-extras-optional-features-with-their-own-dependencies
Example of usage. 
https://github.com/psf/requests/blob/3c1d36b827417fdeaf5a1c106129de30dac371d7/setup.py#L98